addOns/automation/src/main/java/org/zaproxy/addon/automation/jobs/PolicyDefinition.java

```diff
@@ -117,7 +130,32 @@ public static void parsePolicyDefinition(
         }
     }
 
+    void checkAndSetDefault(
```

Comment:
`private static`

Reply:
Not yet addressed.

Reply:
Oops I went too fast, I had thought that was also about the test assertion method 😞

Reply:
Actually done now :wink: